PrototopeJSBridge/LayerBridge.swift

```diff
@@ -76,6 +76,10 @@ import JavaScriptCore
 
     // MARK: Animations
     var animators: LayerAnimatorStoreJSExport { get }
+	
+	// MARK: Particle emitters
+	func addParticleEmitter(emitterBridge: ParticleEmitterBridge)
```

Comment:
Unfortunately the reason these aren't bridging is that you have to type these as `emitterBridge: ParticleEmitterJSExport`, not `ParticleEmitterBridge`. This… took me a while to figure out.